### PR TITLE
Added some getters for LIFTOPT.

### PR DESCRIPTION
**Pull request by hakonhagland**
Added getters for `gaslift_increment()` and `min_eco_gradient()` in the `Opm::GasLiftOpt` class.

### CODE REVIEW

File: opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp [hunk 1]
```diff
@@ -215,7 +215,9 @@ class GasLiftOpt {
     const Group& group(const std::string& gname) const;
     const Well& well(const std::string& wname) const;
 
+    double gaslift_increment();
```
Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp:218]: The get methods should be marked `const`- i.e. 
```
double gaslift_increment() const;
```

**Comment by joakim-hove** [comment]
One minor comment about `const`- otherwise this goes right in!

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by hakonhagland** [comment]
> One minor comment about const

@joakim-hove Of course! Updated.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**